### PR TITLE
Revise crytpo policy show usage (SOFTWARE-5330)

### PR DESCRIPTION
**Pull request by jeff-takaki**
Add an if statement to check for version 8 or 9 because update-crypto-policies --show does not work with EL7

### CODE REVIEW

**Review by matyasselmeci — CHANGES_REQUESTED** [review]
Instead of explicitly checking /etc/os-release for the distro version, you should check the return code of `which update-crypto-policies &>/dev/null` to see if the script itself exists. (`&>XXX` is shorthand for `>XXX 2>&1`, i.e. redirecting both stdout and stderr).

The other thing: osg-system-profiler output is divided into sections. A section is a line that looks like
```
***** SECTION NAME
```
You should print out a new section for the crypto policy check; if you do not, it would be appended to the end of the previous section.

**Comment by jeff-takaki** [comment]
@matyasselmeci  Does that change look better?

File: osg-system-profiler [hunk 1]
```diff
@@ -184,7 +184,14 @@ dump_file "/etc/hosts"
 
 run_cmd df --human-readable --print-type
 run_cmd free
-run_cmd update-crypto-policies --show
+
+
+if which update-crypto-policies &>/dev/null; then
+    echo "***** Crypto Policy"
+    update-crypto-policies --show
```
Comment (matyasselmeci) [inline osg-system-profiler]:
```suggestion
    update-crypto-policies --show 2>&1
```
Let's capture stderr if there is any.

**Review by matyasselmeci — APPROVED** [review]
One suggestion; pre-approving

File: osg-system-profiler (hunk 1 above)
Comment (jeff-takaki) [reply osg-system-profiler]: @matyasselmeci gotcha!

**Pull request merged**